### ISSUE TITLE
MultiInput: change event wouldn't be fired when typed value is as same as the first token 

### ISSUE THREAD

**Issue by echowwww** [SAP/ui5-webcomponents#6262]
### **Bug Description**
If the second input has the same value as the first input, change event wouldn't be fired and user cannot press enter.

### **Expected Behavior**
change event should be fired when user press enter or focus out the input field. In change event listener, user can then check for duplicated input or perform other actions.

### **Steps to Reproduce**
1. Go to ui5's office document and navigate to multi input component with token creation onChange
2. enter Argentina and press enter to create a token
3. enter Argentina again and press enter

### **Context**
- UI5 Web Components version: {v1.9.1}
- OS/Platform: {MacOs}
- Browser: {Microsoft Edge}
- Affected component: {MultiInput}

### **Log Output / Stack Trace / Screenshots**
```
{...}
```
### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

**Comment by kineticjs** [comment]
Hi @echowwww ,

Are you reproducing this on https://sap.github.io/ui5-webcomponents/playground/components/MultiInput/ ,
sample: "Multi Input and token creation onChange"?

I set a breakpoint and see that the "change" event is fired but the logic for keeping *unique* tokens only executes, as shown in the screenshot below:

![duplicatetoken](https://user-images.githubusercontent.com/8408297/211624208-1c3314fd-a239-4da3-ab05-7a5cdbdf0d30.png)

Please comment if you are testing on a different sample.

**Comment by echowwww** [comment]
Yes, i use this on https://sap.github.io/ui5-webcomponents/playground/components/MultiInput/ ,
sample: "Multi Input and token creation onChange". 
I set a breakpoint at line 5989. When enter second 'Argentina' and press enter, breakpoint is not triggered. The warning 'token already in list' not show neither.
The case only happens when the first and second inputs are same. If 'Argentina' is third input, everything works fine.
Just FYI, i use Microsoft Edge, latest version. Not sure whether it's relevant.

**Comment by kineticjs** [comment]
Hi @SAP/ui5-webcomponents-topic-rl,

Can you please check this issue with handling of duplicate input texts in ui5-multiinput:

To reproduce:
1. Open https://sap.github.io/ui5-webcomponents/playground/components/MultiInput/  
sample  "Multi Input and token creation onChange"
2. add a conditional breakpoint in the listener for the "change" event of ui5-multiinput of the same sample
(see the attached screenshot above) with condition: **console.log("change")**
3. Add twice the token "Argentina" **without moving the focus away from the multiinput field**
Expected: The change event is fired (=>logged in the console) twice; 
                and the second time the code should display the value state message 'token already in list'
Actual: The change event is fired only once (for the first token only)

The issue seems to be related to the logic in the _handleChange function of ui5-input, which fires the "change" event only if a precondition with the previously entered value is in place.
